Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
+
+EXPOSE 8000
+WORKDIR /code
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential libxml2-dev libxslt1-dev python-dev \
+    libmysqlclient-dev python-pip node-less && \
+    pip install --upgrade pip && \
```

Comment:
I'd pin a pip version. In general pinning in docker is the way to go, to make sure that you get reproducible builds. Similarly you could pin debian packages but because you tagged the debian release, I take it as good enough. Debian takes care not to make major upgrades to packages of releases therefore things should not break.